src/dbSta/src/dbNetwork.cc

```diff
@@ -962,12 +1061,14 @@ void dbNetwork::makeTopCell()
   }
   const char* design_name = block_->getConstName();
   Library* top_lib = makeLibrary(design_name, nullptr);
+  top_cell_ = nullptr;
   top_cell_ = makeCell(top_lib, design_name, false, nullptr);
```

Comment:
What is the purpose of the first assign?

Reply:
removed